logging/Notes_on_using_TLS.md

```diff
@@ -71,4 +71,21 @@ export USER_DIR=~/my-cluster-files/ops/user-dir
 
    - For the value of the key `opendistro_security.nodes_dn`, specify the subject information that you obtained from the `es-transport-tls` certificate.
 
-*Note:* You must specify the subject information exactly as it was returned from the certificate (without any spaces after the commas). 
\ No newline at end of file
+***Note:*** You must specify the subject information exactly as it was returned from the certificate (without any spaces after the commas). 
+
+7. If you plan on using ingress,populate these Kubernetes secrets with 
+TLS certificates before you run the deployment script:
+
+* `kibana-ingress-tls-secret`
+* `elasticsearch-ingress-tls-secret`
+
+8. After you have obtained the certificates for the secrets, use this command to generate the secrets:
+
+```bash
+kubectl create secret tls $SECRET_NAME -n $NAMESPACE --key $CERT_KEY --cert $CERT_FILE
+```
+
+Use `kibana-ingress-tls-secret` and `elasticsearch-ingress-tls-secret` as values for `$SECRET_NAME`. Use `logging` for the value of `$NAMESPACE`.
+
```

Comment:
They should use whatever the name of namespace they used for the logging components for NAMESPACE...which may or may not be "logging".

Reply:
Revised as suggested